src/software/thunderscope/field/obstacle_layer.py

```diff
@@ -27,20 +32,18 @@ def paint(self, painter, option, widget):
 
         """
 
-        try:
-            obstacles = self.obstacle_buffer.get_nowait()
-        except queue.Empty as empty:
-            obstacles = self.cached_obstacles
-
-        self.cached_obstacles = obstacles
+        # Draw the obstacles
+        obstacles = self.obstacle_buffer.get(block=False)
 
-        painter.setPen(pg.mkPen(Colors.NAVIGATOR_OBSTACLE_COLOR, width=2))
+        painter.setPen(
+            pg.mkPen(Colors.NAVIGATOR_OBSTACLE_COLOR, width=constants.LINE_WIDTH)
```

Comment:
can this be thinner, it kinda blocks out everything

Reply:
done